README.md

```diff
@@ -28,3 +28,153 @@ Tentative roadmap:
+```
+
+Windows example:
+```
+% GOOS=windows go build -C x -o ./bin/ ./outline-connectivity 
+% file ./x/bin/outline-connectivity.exe 
+./x/bin/outline-connectivity.exe: PE32+ executable (console) x86-64 (stripped to external PDB), for MS Windows
+```
+</details>
+
+## Running Linux binaries
+
+To run Linux binaries you can use a Linux container via [Podman](https://podman.io/).
+
+### Set up podman
+<details>
+  <summary>Instructions</summary>
+
+[Install Podman](https://podman.io/docs/installation)
+On macOS (once):
+```sh
+brew install podman
+```
+
+Create the podman service VM (once):
+```sh
+podman machine init
+```
+
+Start the VM (after every time it is stopped):
+```sh
+podman machine start
+``` 
+
+You can see it running with `podman machine list`:
+```
+% podman machine list
+NAME                     VM TYPE     CREATED        LAST UP            CPUS        MEMORY      DISK SIZE
+podman-machine-default*  qemu        3 minutes ago  Currently running  1           2.147GB     107.4GB
+```
+
+When you are done with development, you can stop the machine:
+```sh
+podman machine stop
+```
+</details>
+
+### Run
+
+The easiest way is to run using `go run` directly with the `--exec` flag and our convenience tool `run_on_podman.sh`:
+```sh
+GOOS=linux go run -C x -exec "$(pwd)/run_on_podman.sh" ./outline-connectivity
+```
+
+It also works for tests:
+```sh
+GOOS=linux go test -exec "$(pwd)/run_on_podman.sh"  ./...
+```
+
+<details>
+  <summary>Details and direct invocation</summary>
+
+The `run_on_podman.sh` script uses `podman run` and the minimal [Alpine Linux](https://en.wikipedia.org/wiki/Alpine_Linux) to run the binary you want:
+```sh
+podman run --rm -it -v "${bin}":/outline/bin alpine /outline/bin "$@"
+```
+
+You can also use `podman` directly to run a pre-built binary:
+```
+% podman run --rm -it -v ./x/bin:/outline alpine /outline/outline-connectivity
+Usage of /outline/outline-connectivity:
+  -domain string
+        Domain name to resolve in the test (default "example.com.")
+  -key string
+        Outline access key
+  -proto string
+        Comma-separated list of the protocols to test. Muse be "tcp", "udp", or a combination of them (default "tcp,udp")
+  -resolver string
+        Comma-separated list of addresses of DNS resolver to use for the test (default "8.8.8.8,2001:4860:4860::8888")
+  -v    Enable debug output
+```
+
+Flags explanation:
+- `--rm`: Remove container (and pod if created) after exit
+- `-i` (interactive): Keep STDIN open even if not attached
+- `-t` (tty): Allocate a pseudo-TTY for container
+- `-v` (volume): Bind mount a volume into the container. Volume source will be on the server machine, not the client
+</details>
+
+## Running Windows binaries
+
+To run Windows binaries you can use [Wine](https://en.wikipedia.org/wiki/Wine_(software)) to emulate a Windows environment.
+This is not the same as a real Windows environment, so make sure you test on actual Windows machines.
+
+### Install Wine
```

Comment:
I think another disadvantage of Wine (compared to a VM) is that it does not support Windows 11 yet.